modules/virt-creating-local-block-pv.adoc

```diff
@@ -34,7 +34,7 @@ $ losetup </dev/loop10>d3 <loop10> <1> <2>
 <1> File path where the loop device is mounted.
 <2> The file created in the previous step to be mounted as the loop device.
 
-. Create a `PersistentVolume` configuration that references the mounted loop device.
+. Create a `PersistentVolume` manifest that references the mounted loop device.
```

Comment:
Yeah, we have inconsistency in our docs. It's something I have marked down in my notes from this PR to bring up with the team.
As I understand it, the manifest is the YAML (we default to files but it needn't be) of a resource not-yet created. You 'oc create|apply' it to the cluster and it creates an object based on that manifest. The manifest then is meaningless (although if it's a file you can edit it to create another similar object). After it's created, everything we do it is is 'editing/deleting/verbing' the object. 
To me it makes sense to use the `API-object` markup for a manifest, since it then matches the `kind: ` in the manifest itself, which is more than likely going to be subsequent in the procedure. But maybe this markup is strictly for objects.
@bergerhoffer Do you have guidance on this?

Reply:
I'm not an expert on the terms following the name of the API object (manifest vs. configuration vs. object, etc.), but your explanation sounds reasonable to me. I agree to the use of `PersistentVolume` here (as opposed to persistent volume), since it is an actual object definition. 

I probably do more often see this referred to as an object definition or configuration when saving them to a file before oc apply-ing, but again your explanation of the usage of manifest makes sense to me.